From b9a100f369fcba29dac624358c57739e842b7e68 Mon Sep 17 00:00:00 2001
From: yplam <yplam@yplam.com>
Date: Tue, 11 Jun 2024 19:52:54 +0800
Subject: [PATCH] change version domain

---
 supervisor/const.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/supervisor/const.py b/supervisor/const.py
index c87f8c11..33cdad34 100644
--- a/supervisor/const.py
+++ b/supervisor/const.py
@@ -13,7 +13,7 @@ SERVER_SOFTWARE = f"HomeAssistantSupervisor/{SUPERVISOR_VERSION} aiohttp/{aiohtt
 
 URL_HASSIO_ADDONS = "https://github.com/home-assistant/addons"
 URL_HASSIO_APPARMOR = "https://version.home-assistant.io/apparmor_{channel}.txt"
-URL_HASSIO_VERSION = "https://version.home-assistant.io/{channel}.json"
+URL_HASSIO_VERSION = "https://version.pantherha.com/{channel}.json"
 
 SUPERVISOR_DATA = Path("/data")
 
-- 
2.45.1

